R/Module-Characterization.R

```diff
@@ -26,6 +26,13 @@ CharacterizationModule <- R6::R6Class(
       workFolder <- jobContext$moduleExecutionSettings$workSubFolder
       resultsFolder <- jobContext$moduleExecutionSettings$resultsSubFolder
 
+      # Handle cleaning of incremental folders/files
+      if (isTRUE(executionSettings$incremental)) {
+        Characterization::cleanIncremental(workFolder)
+      } else {
+        Characterization::cleanNonIncremental(workFolder)
+      }
+
```

Comment:
@jreps , could you provide feedback if this is the appropriate use of cleanIncremental vs. cleanNonIncremental?

Reply:
Just looking at the Characterization code, I think I need to update the approach in this branch so that when `executionSettings$incremental == FALSE`, it calls `Characterization::cleanNonIncremental(resultsFolder)` instead of `Characterization::cleanNonIncremental(workFolder)`. 

What is still unclear to me is: when `executionSettings$incremental == FALSE`, does the module need to do anything to the `workFolder`? Should it always call `Characterization::cleanIncremental(workFolder)`?